Tests/pageObject/RadioNavigationalWarningsAdminListEndUser.page.ts

```diff
@@ -179,4 +186,32 @@ export default class RadioNavigationalWarningsListEndUser {
 
   }
  
+  public async verifyAboutrnw()
+  {
+    await this.aboutEndUser.click();
+    expect(await this.about.evaluate(option => option.getAttribute('href'))).toContain('https://iho.int/navigation-warnings-on-the-web')
+  }
+
+  public async verifyAboutRNWImportantBlock() {
```

Comment:
This method is almost identical to the function verifyImportantBlock()
Please refactor to remove repeated code.

Reply:
resolved